src/hooks/useSubaccount.tsx

```diff
@@ -293,12 +293,29 @@ export const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: Lo
   );
 
   const sendSquidWithdraw = useCallback(
-    async (amount: number, payload: string) => {
+    async (amount: number, payload: string, isCCtp?: boolean) => {
```

Comment:
`isCCtp` -> `isCCTP`? why not capitalize transfer protocol

Reply:
Good catch, I'll keep it isCctp to be consistent with abacus